src/tir/transforms/make_packed_api.cc

```diff
@@ -120,6 +120,11 @@ PrimFunc MakePackedAPI(PrimFunc&& func, int num_unpacked_args) {
   int num_args = static_cast<int>(func_ptr->params.size());
   ICHECK_LE(num_unpacked_args, num_args);
 
+  bool pack_args = true;
+  if (num_unpacked_args == -1) {
```

Comment:
Can we also update the python side and update the documents about this behavior>